pkg/controller/controller_test.go

```diff
@@ -565,6 +594,15 @@ var _ = Describe("CSIControllerService DeleteVolume", func() {
 			err = controller.k8sclient.CreateCR(testCtx, volumeID, volumeCrd)
 			Expect(err).To(BeNil())
 
+			ac := testAC1.DeepCopy()
+			ac.Spec.Size = 0
+			err = controller.k8sclient.CreateCR(testCtx, testAC1Name, ac)
```

Comment:
`Expect(controller.k8sclient.CreateCR(testCtx, testAC1Name, ac)).To(BeNil())`
No need to `err` variable assigment.

Reply:
ginkgo stop test execution after failed `Expect` and it's more comfortable for me to have separate lines for these expressions for debugging. It gives an option to put a breakpoint on a line with `createCR` and check stack trace and so on